### ISSUE TITLE
Links not rendered correctly

### ISSUE THREAD

**Issue by newmedicine** [secure-77/Perlite#1]
**Image versions:**
```
REPOSITORY                     TAG          IMAGE ID       CREATED         SIZE
perlite_web                    latest       9ce1a0f8db97   4 minutes ago   133MB
perlite_perlite                latest       e280f9861c9d   5 minutes ago   469MB
```

**ObsidianFile.md:**
```
- [[Staph. aureus]]
- [[Pseudomonas aeruginosa]]
- [[Stenotrophomonas maltophilia]]
- [[Burkholderia cepacia complex]]
- [[Achromobacter xylosoxidans]]
- [[Non-Tuberculous Mycobacteria (NTM)]]
- [[Fungal pathogens]]
```

**Rendered as:**
![Cursor_and______Perlite_____](https://user-images.githubusercontent.com/22443747/131216214-45b7dcbf-099b-40d9-bb96-13a897a1252e.png)

Am I doing something wrong?

Thanks

Nick

**Comment by secure-77** [comment]
Hey Nick, 
Thanks for your issue. As in the readme described:
```
unfortunately Obsidian links to other files then images will not work at the moment (feature is planed)
```

So only obsidian links to images and real website links are working at the moment. As soon as i have some time i will try to implement this feature.

BR,
Sec77

**Comment by newmedicine** [comment]
Ah OK RTFM sorry.

**Comment by secure-77** [comment]
Hey @newmedicine,
i released Version 1.3 which support links to other markdown and PDF files. Would be nice to get a feedback if this fixes your problem.

BR,
Sec77
